go-controller/pkg/ovn/master.go

```diff
@@ -425,24 +425,24 @@ func addStaticRouteToHost(node *kapi.Node, nicIPs []*net.IPNet) error {
 	if prefix == "" {
 		return fmt.Errorf("configuration error: no NIC IP of same family as hostsubnet")
 	}
-	nexthop := secondIP.IP.String()
+	nexthop := mgmtPortAddr.IP.String()
 	_, stderr, err := util.RunOVNNbctl("--may-exist", "lr-route-add", k8sClusterRouter, prefix, nexthop)
 	if err != nil {
 		return fmt.Errorf("failed to add static route '%s via %s' for host %q on %s "+
-			"stderr: %q, error: %v", prefix, secondIP, node.Name, k8sClusterRouter, stderr, err)
+			"stderr: %q, error: %v", prefix, mgmtPortAddr, node.Name, k8sClusterRouter, stderr, err)
 	}
 
 	return nil
 }
 
 func (oc *Controller) ensureNodeLogicalNetwork(nodeName string, hostsubnet *net.IPNet) error {
-	firstIP, secondIP := util.GetNodeWellKnownAddresses(hostsubnet)
-	nodeLRPMAC := util.IPAddrToHWAddr(firstIP.IP)
+	gwAddr, mgmtPortAddr, hybridOverlayAddr := util.GetNodeWellKnownAddresses(hostsubnet)
```

Comment:
it's more specificlaly `hybridOverlayGWAddr`, right? (likewise below in util.go)

Reply:
Updated to "hybridOverlayIfAddr" per above.